src/services/auth/plugins/mobile/schemas.ts

```diff
@@ -3,10 +3,31 @@ import { StatusCodes } from 'http-status-codes';
 
 import { FastifySchema } from 'fastify';
 
-import { customType } from '../../../../plugins/typebox';
+import { customType, registerSchemaAsRef } from '../../../../plugins/typebox';
+import { errorSchemaRef } from '../../../../schemas/global';
 import { SHORT_TOKEN_PARAM, TOKEN_PARAM } from '../passport';
 
+const authTokensPairSchemaRef = registerSchemaAsRef(
+  'tokensPair',
+  'Tokens Pair',
+  Type.Object(
+    {
+      authToken: Type.String(),
+      refreshToken: Type.String(),
+    },
+    {
+      // Schema Options
+      additionalProperties: false,
```

Comment:
It is only a recommandation, but we can add a small description to each `SchemaRef`. [Source](https://quobix.com/vacuum/rules/descriptions/component-description/)